docs/ops/movement/StridedSlice_1.md

```diff
@@ -51,19 +51,19 @@
 
 **Inputs**:
 
-*   **1**: Multidimensional input tensor to be sliced. Required.
+*   **1**: input tensor to be sliced of type `T` and arbitrary shape. **Required.**
 
-*   **2**: `begin` input - 1D input tensor with begin indexes for input tensor slicing. Required.
-           Out-of-bounds values are silently clamped. If `begin_mask[i]` is 1, the value of `begin[i]` is ignored
-           and the range of the appropriate dimension starts from 0.
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
+*   **2**: `begin` - 1D tensor of type `T_IND` with begin indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values are silently clamped. If `begin_mask[i]` is `1`, the value of `begin[i]` is ignored and the range of the appropriate dimension starts from `0`. Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
 
-*   **3**: `end` input - 1D input tensor with end indexes for input tensor slicing. Required.
-           Out-of-bounds values will be silently clamped. If `end_mask[i]` is 1, the value of `end[i]` is ignored
-           and the full range of the appropriate dimension is used instead.
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `end[0]=-1` means `end[0]=3`.
+*   **3**: `end` - 1D tensor of type `T_IND` with end indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values will be silently clamped. If `end_mask[i]` is `1`, the value of `end[i]` is ignored and the full range of the appropriate dimension is used instead. Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `end[0]=-1` means `end[0]=3`.
 
-*   **4**: `stride` input - 1D input tensor with strides. Optional.
+*   **4**: `stride` - 1D tensor of type `T_IND` with strides. **Optional.**
+
+**Types**
+* *T*: any supported floating point type.
```

Comment:
It is not only floating, but integer and boolean as well.

Reply:
In terms of operator class this statement seems correct, but there are only [FP32 SLT tests for CPU plugin](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/strided_slice.cpp#L123) so how do you know if it works also for other types?

Reply:
I've changed it to 'any supported type' in this [commit](https://github.com/openvinotoolkit/openvino/pull/5887/commits/b505861342b567b88f9e36323a85038d60b34a46).